lib/pluggable.js

```diff
@@ -0,0 +1,115 @@
+// TODO: make DRY
+
+import * as most from "most";
+
+
+export const CONTINUE = Symbol.for("interlock.continue");
+
+
+function getContext (baseCxt, dependencies) {
```

Comment:
Side effects?  It takes two inputs, and produces a single output.  Is it just the `get` part that you object to?  I could call it `generateContext` if that's better.